src/main/kotlin/io/github/mojira/risa/application/ReportGenerator.kt

```diff
@@ -14,7 +14,7 @@ fun generateReport(
 ): Report = buildString {
     append("[Mojang's Release Post](https://www.minecraft.net/en-us/article/minecraft-snapshot-${currentSnapshot.name})")
     append(" ~ [Last Report](https://www.reddit.com/r/Mojira/comments/${snapshotPosts.getPreviousOf(currentSnapshot)})")
-    append("Last updated: ${Instant.now()}")
+    append(" Last updated: ${Instant.now()}")
```

Comment:
```suggestion
    append(" ~ Last updated: ${Instant.now()}")
```
Might make sense to use the same separator as for the previous line.

Reply:
Also perhaps the date could be formatted a little better? The THH:MM:SS.xxxZ string isn't easy to read.

Reply:
Its the iso format, the one that should cause the less confusion :eyes:

Reply:
I'd disagree. If it'd say `2021-04-08 16:00 UTC` or something that would both be easier to read and cause less confusion. ISO is easy for computers to parse, but not for regular users. But perhaps that'd be a change for a different PR.

Reply:
I would put it in a differnt one